cvmfs/cvmfs.cc

```diff
@@ -1900,7 +1903,6 @@ bool Pin(const string &path) {
         return false;
       }
       cache_manager_->Close(fd);
-      return true;
```

Comment:
Oh, I see.  I looked at the source and thought I had traced it back to an if, my mistake.